### PR TITLE
Continue

### PR DESCRIPTION
**Pull request by MarvinTeichmann**

### CODE REVIEW

File: bin/tv-continue [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Trains, evaluates and saves the model network using a queue."""
```
Comment (MartinThoma) [inline bin/tv-continue]: Please adjust this.

File: bin/tv-continue [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+flags = tf.app.flags
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string('logdir', None,
+                    'Directory where logs are stored.')
+
+
+def main(_):
+    """Run main function for continue Training."""
```
Comment (MartinThoma) [inline bin/tv-continue]: "training", not "Training"

File: bin/tv-continue [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    if FLAGS.logdir is None:
+        logging.error("No logdir are given.")
```
Comment (MartinThoma) [inline bin/tv-continue]: "No logdir is given."

**Comment by MartinThoma** [comment]
I'm a bit confused now... wasn't this also the main contribution of #39?

**Comment by MarvinTeichmann** [comment]
No, #39 is one commit ahead of #38. Did this to avoid merge conflicts. (i.e. used checkout -b continue before starting #39)

**Comment by MartinThoma** [comment]
Ah, ok. Then I guess I'm ok with merging this pull request, too.

**Pull request closed without merging**